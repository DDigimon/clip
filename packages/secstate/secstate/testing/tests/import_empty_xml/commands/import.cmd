/usr/bin/secstate import /root/empty.xml
