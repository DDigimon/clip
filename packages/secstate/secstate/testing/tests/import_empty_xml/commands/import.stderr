Error parsing '/root/empty.xml': no element found: line 1, column 0
